strawberryfields/plot.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This module provides tools to visualize the state in various interactive
+ways using Plot.ly
+"""
+from copy import deepcopy
+
+import numpy as np
+import plotly.io as pio
```

Comment:
We need to be careful with the import here, since plotly isn't a dependency of SF! If you have a look in `sf.apps.plot`, you can see how this is handled there (essentially, lots of try excepts)